packages/server/test/manifest-builder.test.ts

```diff
@@ -177,9 +177,37 @@ describe('manifest builder', () => {
       // assert is used to type narrow also and does more than just assert
       assertBundlerState(bundlerState.type, {is: 'ERRORED'})
 
-      let error = bundlerState.error;
+      expect(bundlerState.error.message).toEqual('bork')
+    });
+  });
 
-      expect(error.message).toEqual('bork')
+  describe('importing a test with no default export adds the error to the state', () => {
+    beforeEach(async () => {
+      await copyFile(path.join(FIXTURES_DIR, 'exceptions', 'no-default-export.t.js'), MANIFEST_PATH);
+      await actions.fork(atom.slice('bundler').once(({ type }) => type === 'ERRORED'));
     });
-  })
+
+    it('should update the global state with the error detail', () => {
+      let bundlerState = atom.get().bundler;
+
+      assertBundlerState(bundlerState.type, {is: 'ERRORED'})
+
+      expect(bundlerState.error.message).toContain('default export')
+    });
+  });
+
+  describe('importing an invalid test object adds the error to the state', () => {
+    beforeEach(async () => {
+      await copyFile(path.join(FIXTURES_DIR, 'exceptions', 'invalid-test-object.t.js'), MANIFEST_PATH);
+      await actions.fork(atom.slice('bundler').once(({ type }) => type === 'ERRORED'));
+    });
+
+    it('should update the global state with the error detail', () => {
+      let bundlerState = atom.get().bundler;
+
+      assertBundlerState(bundlerState.type, {is: 'ERRORED'})
+
+      expect(bundlerState.error.message).toContain('Test contains no assertions or children')
+    });
+  });
```

Comment:
There are tests specific for `validateTest` in the suite package here: https://github.com/thefrontside/bigtest/blob/8ffcba281845b2f7bc0334c40718f53a68386b1b/packages/suite/test/validate-test.test.ts

I think it would make sense to add those tests there?

Reply:
@jnicklas I agree, that is a better place.